README.md

```diff
@@ -37,6 +37,46 @@ Libfabric distribution tarballs are already bootstrapped for you.
 
 Libfabric currently supports GNU/Linux, Free BSD, and OS X.
 
+## WINDOWS Instructions
+Even though windows isn't fully supported yet it is possible to compile and link your library.
+- 1. first you need the NetDirect provider:
+  Network Direct SDK/DDK may be obtained as a nuget package (preferred) from:
+
+  https://www.nuget.org/packages/NetworkDirect
+
+  or downloaded from:
+
+  https://www.microsoft.com/en-us/download/details.aspx?id=36043
+  on page press Download button and select NetworkDirect_DDK.zip.
+
+  Extract header files from downloaded
+  NetworkDirect_DDK.zip:`\NetDirect\include\` file into `<libfabricroot>/prov/netdir/NetDirect/`,
```

Comment:
`<libfabricroot>/prov/netdir/NetDirect/` -> `<libfabricroot>\prov\netdir\NetDirect\`
and please, update `/` -> `\` below